src/Storage/Helpers/DataElementHelper.cs

```diff
@@ -1,8 +1,7 @@
 using System;
 using System.Collections.Generic;
-using System.Linq;
-
 using Altinn.Platform.Storage.Interface.Models;
+using Microsoft.IdentityModel.Tokens;
```

Comment:
Doesn't seem this using is required for the new functionality?

Reply:
I used it to check if generatedFromTask is null or empty, but can rewrite check to use string.IsNullOrEmpty(generatedFromTask) instead :)